debezium-core/src/test/java/io/debezium/relational/TableSchemaBuilderTest.java

```diff
@@ -73,6 +76,14 @@ public void beforeEach() {
                                        .type("SMALLINT").jdbcType(Types.SMALLINT)
                                        .optional(false)
                                        .length(1)
+                                       .create(),
+                                Column.editor().name("7C7") // test invalid Avro name (starts with digit)
+                                       .type("VARCHAR").jdbcType(Types.VARCHAR).length(10)
+                                       .optional(false)
+                                       .create(),
+                                 Column.editor().name("C-8") // test invalid Avro name ( contains dash )
```

Comment:
Could you please add a test column that will contain all invalid characters not supported by Avro?